WebCryptoAPI/util/helpers.js

```diff
@@ -160,6 +160,7 @@ function assert_goodCryptoKey(key, algorithm, extractable, usages, kind) {
         assert_in_array(usage, correctUsages, "Has " + usage + " usage");
     });
     assert_equals(key.usages.length, usageCount, "usages property is correct");
+    assert_equals(Object.prototype.toString.call(key), '[object CryptoKey]', "has the expected Symbol.toStringTag");
```

Comment:
Given the name of the test, I would explicitly test that
```suggestion
    assert_equals(key[Symbol.toStringTag], 'CryptoKey', "has the expected Symbol.toStringTag");
```
(either instead, or additionally - but pedantically speaking the behavior of `Object.prototype.toString` given the existence of that `Symbol.toStringTag` property can be left to be tested elsewhere, imo).